devel/readline/Portfile

```diff
@@ -4,8 +4,8 @@ PortGroup       muniversal 1.0
 PortGroup       clang_dependency 1.0
 
 name            readline
-set milestone   8.1.2
-set patchlevel  000
+set milestone   8.2
```

Comment:
I suspect that the library name will have changed with this update? The current version provides for example `/opt/local/lib/libreadline.8.1.dylib` in addition to `/opt/local/lib/libreadline.8.dylib` and `/opt/local/lib/libreadline.dylib`.

Not sure which of those other packages will link against - but I suspect this update might require some `revision` bumps for other ports as well? Or did you verify that other ports that depend on `readline` still work?

Reply:
I tested it out with postgres' psql locally which worked fine. Also ran port reclaim to get it to do the system-wide linker checks (I had a few other things using it) and it didn't turn up anything.

I forget what otool -L on psql had (away from machine for a bit to check) but remember being surprised - it may have been linking against the unversioned readline symlink.

And not that it should count for much but past pull requests weren't revbumping the universe.

Reply:
```
% otool -D /opt/local/lib/libreadline.8.1.dylib
/opt/local/lib/libreadline.8.1.dylib:
/opt/local/lib/libreadline.8.dylib
```
No major library version change.